lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1463,19 +1454,68 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => true,
           :description => 'Distributed tracing lets you see the path that a request takes through your distributed system. Enabling distributed tracing changes the behavior of some New Relic features, so carefully consult the [transition guide](/docs/transition-guide-distributed-tracing) before you enable this feature.'
         },
+        :'distributed_tracing.sampler.root' => {
+          :default => 'default',
+          :public => true,
+          :type => String,
+          :allowed_from_server => false,
```

Comment:
Don't know if these should or shouldn't be allowed

Reply:
@tannalynn - 📰 None of these configs are expected to be allowed from the server.